src/lightbox-react.js

```diff
@@ -509,7 +525,8 @@ class LightboxReact extends Component {
      * Handle user keyboard actions
      */
     handleKeyInput(event) {
-        event.stopPropagation();
+        // this will prevent key up event in order to avoid that commented this
+        // event.stopPropagation();
```

Comment:
What's the use case for allowing the key input to propagate?

Reply:
This was done to allow screen readers to read by key boards navigation(which will make the light box accessible)